libsplinter/src/registry/client/mod.rs

```diff
@@ -0,0 +1,110 @@
+// Copyright 2018-2021 Cargill Incorporated
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! Traits and implementations useful for communicating with the registry as a client.
+
+use std::fmt;
+
+mod reqwest;
```

Comment:
This should be protected by a feature called "registry-client-reqwest".

Reply:
Fixed